lib/trino-orc/src/main/java/io/trino/orc/OrcWriterOptions.java

```diff
@@ -74,7 +75,8 @@ public OrcWriterOptions()
                 DEFAULT_MAX_STRING_STATISTICS_LIMIT,
                 DEFAULT_MAX_COMPRESSION_BUFFER_SIZE,
                 ImmutableSet.of(),
-                DEFAULT_BLOOM_FILTER_FPP);
+                DEFAULT_BLOOM_FILTER_FPP,
+                false);
```

Comment:
Is this the default? i think it should be true

Reply:
hm ok I can change this but I didn't want to change current behaviour for all connectors that are using trino-orc

Reply:
The new behavior is just better